src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -215,68 +310,79 @@ const DiffExprCompute = (props) => {
       }} defaultValue={selectedComparison}>
         <Radio
           style={radioStyle}
-          value={ComparisonType.within}>
+          value={ComparisonType.WITHIN}>
           Compare cell sets within a sample/group
         </Radio>
         <Radio
           style={radioStyle}
-          value={ComparisonType.between}
+          value={ComparisonType.BETWEEN}
           disabled={!hasMetadata || numSamples === 1}
         >
           Compare a selected cell set between samples/groups
         </Radio>
       </Radio.Group>
 
-      {selectedComparison === ComparisonType.between
+      {selectedComparison === ComparisonType.WITHIN
```

Comment:
I switch the order here so that it reflects the order shown in the UI - within first, and then between.